src/Data/Product/Relation/Binary/Lex/Strict.agda

```diff
@@ -202,13 +202,13 @@ module _ {_≈₁_ : Rel A ℓ₁} {_<₁_ : Rel A ℓ₂} {_<₂_ : Rel B ℓ
     ×-acc : ∀ {x y} →
             Acc _<₁_ x → Acc _<₂_ y →
             WfRec _<ₗₑₓ_ (Acc _<ₗₑₓ_) (x , y)
-    ×-acc (acc rec₁) acc₂ (u , v) (inj₁ u<x)
-      = acc (×-acc (rec₁ u u<x) (wf₂ v))
-    ×-acc acc₁ (acc rec₂) (u , v) (inj₂ (u≈x , v<y))
+    ×-acc (acc rec₁) acc₂ {u , v} (inj₁ u<x)
+      = acc (×-acc (rec₁ {u} u<x) (wf₂ v))
```

Comment:
Instance 2/8, but in fact, this one is inferrable too... I left it in, for 'documentation', but more than happy to eliminate!